README.md

```diff
@@ -1,10 +1,60 @@
 
-# How tos
 
-## Pre-requisites to run tools locally
+# Improving IBM Quantum & Qiskit Documentation
+Maintaining up to date documentation is a huge challenge for any software project, especially in a field like quantum computing where the pace at which new research and technological capabilities get released is incredibly fast. As a result, we greatly appreciate any who take the time to support us in keeping this content accurate and up to the highest quality standard possible to benefit the broadest range of users.
+
+Read on for more information about how to support this project:
+
+## Best ways to contribute to Documentation
+### 1. Report bugs, inaccuracies or general content issues
+
+This is the quickest, easiest and most helpful way to contribute to this project and improve the quality of Qiskit and IBM Quantum documentation. There are a few different ways to report issues depending on where it was found:
+
+- For problems you've found in the [Qiskit API Reference](https://docs.quantum-computing.ibm.com/api/qiskit) section, open an issue in the Qiskit repo [here](https://github.com/Qiskit/qiskit/issues/new/choose)
+- For problems you've found in the [Qiskit Runtime IBM Client](https://docs.quantum-computing.ibm.com/api/qiskit-ibm-runtime/runtime_service) section, open an issue in the Qiskit IBM Runtime repo [here](https://github.com/Qiskit/qiskit-ibm-runtime/issues/new/choose)
+- For problems you've found in any other section of [docs](https://docs.quantum-computing.ibm.com), open a content bug issue [here](https://github.com/Qiskit/documentation/issues/new/choose).
+
+### 2. Suggest new content
+
+If you think there are gaps in our documentation, or sections that could be expanded upon, we invite you to open a new content request issue [here](https://github.com/Qiskit/documentation/issues/new/choose). 
+
+Not every new content suggestion is a good fit for docs, nor are we able to prioritise every request immediately. However, we will do our best to respond to content requests in a timely manner, and we greatly appreicate our community's efforts in generating new ideas.
+
+If you are interested in writing the new content yourself, or already have some draft work you think could be integrated, please also mention that in the issue description. If your content suggestion is accepted we will let you know and work with you to get the content written and reviewed.
+
+Please note: we DO NOT accept unsolicited PRs for new pages or large updates to existing content. The content that we include in docs is carefully planned and curated by our content team and must go through the appropriate review process to ensure the quality is of the highest possible standard before deploying to production. As a result we are very selective with which content suggestions are approved and it is unlikely that PRs submitted without an associated approved content request will be accepted.
+
+### 3. Validate exisiting issues
```

Comment:
```suggestion
### 3. Validate existing issues
```

Reply:
damn you can really tell I wrote this page in a rush can't you lol